client/src/modules/stock/dashboard/dashboard.html

```diff
@@ -56,7 +56,7 @@
     <div class="row">
       <div class="col-md-4">
         <bh-stock-panel-minimum-reached
-          label="STOCK.STATUS.MINIMUM">
+          label="STOCK.DASHBOARD.LOW_STOCK">
```

Comment:
Hey!  Nice! :+1:

We already have LOW_STOCK translated in `STOCK.STATUS.MINIMUM` ([here](https://github.com/IMA-WorldHealth/bhima/blob/00bd25f303b883582881d84df65bca613213f3dc/client/src/i18n/en/stock.json#L182)).  Since this dashboard is basically just looking for stock lots that have a particular flag, it might make sense to just use that translation key here.

I would propose holding back on this change for another PR where we can tackle all references to the stock status flags together.  I'll make an issue on it.

Reply:
I did this deliberately because the "MINIMUM" one was used as a column header, but I wanted something less terse and matching the style of the other Dashboard panes.  So I do not plan to change this unless you really think it is necessary.